### DISCUSSION TITLE
Adding Animation (.glb) file in my site

### DISCUSSION THREAD

**Discussion by Tha1ini** [squidfunk/mkdocs-material#6012] [Q&A]
I used  Model Viewer for displaying GLB files.  The site is rendered. However, the animation file is not embedded. Its just blank

I am not able to understand what I am missing.

**Details :**  

pendulum.md file (location ../docs/pendulum.md)
and has the following code

`<model-viewer src="assets/mechanics/DoublePendulum.glb"
                 alt="Double Pendulum Model"
                 autoplay
                 auto-rotate
                 camera-controls
                 background-color="#f0f0f0">
</model-viewer>`

DoublePendulum.glb is located in (..docs/assets/mechanics/DoublePendulum.glb)

In mkdocs.yml file, added this

```
extra_javascript:
  - "https://unpkg.com/model-viewer"
```

**Comment by alexvoss** [comment]
Hi,

I just had a look and the URL referenced in your `extra_javascript` produces "Cannot find package model-viewer@latest". That would explain it, no? What documentation for model-viewer are you going by?

Cheers,

Alex
